server/utils/getMONGO_URI.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+const getMONGO_URI = () => {
+  try {
+    if (process.env.MONGO_URI === undefined)
```

Comment:
if (!process.env.MONGO_URI)

Reply:
Switched to that in `jwt-authentication-login-page`.